mystic.py

```diff
@@ -4,30 +4,18 @@
 from datetime import datetime, timedelta
 import unidecode
 from bs4 import BeautifulSoup
-from stone import ID_TOKEN, TIEBREAKERS
-
-# TODO(Alex.R) Move this information into stone
-# start_time = datetime(2019, 1, 15)
-#start_time = datetime(2019, 5, 28)
-start_time = datetime(2020, 1, 22)
-WEEKS = [start_time + timedelta(weeks=i) for i in range(20)]
-
-
-def get_week(ts):
-    res = False
-    for i, week in enumerate(WEEKS):
-        if ts < week:
-            res = i
-            break
-    if res:
-        if res == 8:
+from stone import ID_TOKEN, TIEBREAKERS, START_DATE, WEEK_OVERRIDES, OFF_WEEK
+
+start_time = datetime.strptime(START_DATE, '%Y-%m-%d')
+
+def get_week(ts, game_id=None):
+    res = WEEK_OVERRIDES.get(game_id, (ts - start_time).days//7 + 1)
```

Comment:
This grabs the override if there is one, or just uses a calculation for the week offset instead of the enumeration if there is no override